packages/web-components/src/mixins/focusable-no-offset.css

```diff
@@ -0,0 +1,13 @@
+/* From the USWDS styles: */
+/*   https://github.com/uswds/uswds/blob/3dc296ec56cd621fe52d918701fd94621d96a198/src/stylesheets/core/mixins/_focus.scss#L12-L13 */
+button:not([disabled]):focus,
+select:not([disabled]):focus,
+a:not([disabled]):focus,
+h1:focus,
+input:not([disabled]):focus,
+textarea:not([disabled]):focus,
+#form-question
+[role='option']:focus,
+*[tabindex]:focus {
+  outline: 2px solid var(--vads-color-action-focus-on-light);
+}
```

Comment:
> [!note]
> For future focus state work, I would be interested in exploring a solution that is a customizable mixin. Since we will be looking at this in the near future, I am not considering this a blocker for this PR.